resources/bin/ballerina

```diff
@@ -92,7 +92,7 @@ then
       echo "Compatible JRE 8 not found. Please follow the instructions in <BALLERINA_HOME>/INSTALL.txt to install and setup Ballerina."
       exit 1;
     fi
-    $JAVA_COMMAND -jar $CURRENT_PATH/../lib/ballerina-command-${ballerina.command.version}.jar "$@"
+    $JAVA_COMMAND -jar -Dcentos.user=$USER $CURRENT_PATH/../lib/ballerina-command-${ballerina.command.version}.jar "$@"
```

Comment:
Can we use a generic param name for this ie. env.user

Reply:
yes, will rename it